sympy/matrices/common.py

```diff
@@ -1888,6 +1888,7 @@ def subs(self, *args, **kwargs):  # should mirror core.basic.subs
         """
         return self.applyfunc(lambda x: x.subs(*args, **kwargs))
```

Comment:
You still need to use iterable here because the change that happens in `subs` won't affect the args here.

Reply:
yes . Ofcourse.